src/picongpu/submit/pizdaint-cscs/SLURM_Tutorial.md

```diff
@@ -0,0 +1,27 @@
+SLURM examples
```

Comment:
I can't use a symlink because there are small differences in. e.g. hyoerthreading activation, one paramter is different.

Ok will add an alias